File: Source/Tests/ForAlternetStudio/Resources/Debugger/CS/HelloWorld/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace HelloWorld
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        [STAThread]
        static void Main1()
        {
            Application.EnableVisualStyles();
            Application.SetCompatibleTextRenderingDefault(false);
            Application.Run(new Form1());
        }

        private void button1_Click(object sender, System.EventArgs e)
        {
            Close();
        }
    }
}

File: Source/Tests/ForAlternetStudio/Resources/Designer/WinForms-dotnetcore/VB/Form1.vb
Partial Public Class Form1
    Inherits System.Windows.Forms.Form

    Public Sub New()
        MyBase.New
        Me.InitializeComponent
    End Sub

    <System.STAThread()> _
    Public Shared Sub Main()
        System.Windows.Forms.Application.EnableVisualStyles
        System.Windows.Forms.Application.SetCompatibleTextRenderingDefault(False)
        System.Windows.Forms.Application.Run(New Form1())
    End Sub

    Private Sub CloseButton_Click(sender As Object, e As System.EventArgs) Handles CloseButton.Click
        Close
    End Sub
End Class
